tutorials/launchTutorials.m

```diff
@@ -38,7 +38,7 @@
     changeCobraSolver('glpk')
     sparseLP_example;
     uniformSampling;
-    tutorial_IO;
+    tutorial_IO;    
```

Comment:
May you also include `ModelCreation` and `ModelManipulation` here? If you can, it would be nice if you actually could rename the tutorials as `tutorial_modelCreation` and `tutorial_modelManipulation`.